FountainRetrofit.md

```diff
@@ -14,18 +14,31 @@ FountainRetrofit.createNetworkListing(
 )
 ```
 
-There's only one required structure, [`RetrofitNetworkDataSourceAdapter<out ListResponse<Value>>`](RetrofitNetworkDataSourceAdapter.md), which Fountain uses to handle the paging.
+There's only one required structure, [`RetrofitNetworkDataSourceAdapter<out ListResponse<Value>>`], which Fountain uses to handle the paging.
 
 In addition, there are some optional parameters that you can define:
 - `firstPage: Int`: The initial page number, by default its value is 1.
 - [`ioServiceExecutor : Executor`](https://developer.android.com/reference/java/util/concurrent/Executor): The executor with which the service call will be made. By default, the library will use a pool of 5 threads.
 - [`pagedListConfig: PagedList.Config`](https://developer.android.com/reference/android/arch/paging/PagedList.Config): The paged list configuration.
 In this object you can specify several options, for example the `pageSize` and the `initialPageSize`. 
 
+## Network Support Listing Creator for not paged endpoints
 
-## Cache + Network Support Listing Creator
+A Listing with Network support for paged not endpoints can be created invoking `createNetworkListing`.
+```kotlin
+FountainRetrofit.createNotPagedNetworkListing(
+  notPagedRetrifitPageFetcher: NotPagedRetrifitPageFetcher<out ListResponse<out NetworkValue>>,
+  ioServiceExecutor: Executor = FountainConstants.NETWORK_EXECUTOR
+)
+```
+
+There's only one required structure, [`NotPagedRetrifitPageFetcher<out ListResponse<Value>>`], which Fountain uses to handle the paging.
+
+Additionally you can specify the [`ioServiceExecutor : Executor`](https://developer.android.com/reference/java/util/concurrent/Executor). It's executor with which the service call will be made. By default, the library will use a pool of 5 threads.
```

Comment:
The `the` before `executor` is missing. However, I think you can say `It's the executor that'll be used to make the service call`.

Reply:
```suggestion
Additionally you can specify the [`ioServiceExecutor : Executor`](https://developer.android.com/reference/java/util/concurrent/Executor). It's the executor that'll be used to make the service call.
```